cmake/HPX_AddExecutable.cmake

```diff
@@ -121,7 +121,9 @@ macro(add_hpx_executable name)
       get_filename_component(extension ${source} EXT)
       if(${extension} STREQUAL ".cu")
         message(${extension})
-        SET_SOURCE_FILES_PROPERTIES(${source} PROPERTIES LANGUAGE CXX)
+        SET_SOURCE_FILES_PROPERTIES(${source} PROPERTIES
+          LANGUAGE CXX
+          COMPILE_FLAGS ${HPX_CUDA_CLANG_FLAGS})
       endif()
```

Comment:
Sure, but the .cu file could be part of a shared or a static library. So adding it to setup_target would enable the required flags for those cases as well, not only for compiling a .cu file into an executable.